src/types/basic.ts

```diff
@@ -82,6 +82,26 @@ export const NumericIdentifier = new t.Type<number, StringOrNumber>(
 );
 export type NumericIdentifier = t.TypeOf<typeof NumericIdentifier>;
 
+/**
+ * A DoubleQuotedString is a string that is enclosed in double quotes
+ */
+export const DoubleQuotedString = new t.Type<string, string>(
```

Comment:
If this is only used by hostconfig, why add it here? I think we should try think of modules as independent as much as we can